package.json

```diff
@@ -26,7 +28,8 @@
     "link-parent-bin": "^1.0.0",
     "netlify-cli": "^2.6.0",
     "prettier": "^1.15.3",
-    "prettier-eslint-cli": "^4.7.1"
+    "prettier-eslint-cli": "^4.7.1",
+    "typescript": "^3.3.3333"
```

Comment:
All of these deps are added at the root level because eslign plugins need to be installed in the same location as eslint. It would be the same if eslint were global; the plugins would need to be global.

Typescript is added here because otherwise the plugin would use an older version of Typescript which is not actually supported by the plugin anymore.